libs/cri-api-service/src/main/java/uk/gov/di/ipv/core/library/criapiservice/CriApiService.java

```diff
@@ -131,12 +123,8 @@ private BearerAccessToken fetchAccessToken(
                                 errorResponse.getErrorObject(),
                                 new ErrorObject("unknown", "unknown"));
                 LOGGER.error(
-                        "Failed to exchange token with credential issuer with ID '{}' at '{}'. Code: '{}', Description: {}, HttpStatus code: {}",
-                        criId,
-                        criConfig.getTokenUrl(),
-                        errorObject.getCode(),
-                        errorObject.getDescription(),
-                        errorObject.getHTTPStatusCode());
+                        LogHelper.buildErrorMessage(
+                                "Failed to exchange token with credential issuer", errorObject));
```

Comment:
The CRI ID is automatically added to all log messages by the handler, so there's no need to repeat it - and it turns out this was the only use of `criId` here!